src/lib/components/NavMenuTabItem.svelte

```diff
@@ -24,7 +24,8 @@
 <button
     on:click={handleClick}
     class="group active border-none stroke-black"
-    data-app-insights-event-name={`footer-nav-tab-${selectedTab}-clicked`}
+    data-app-insights-event-name="footer-nav-tab-clicked"
+    data-app-insights-dimensions={`label,${label}`}
```

Comment:
Should this one be `selectedTab`?

Reply:
nah, the label works just as fine here. 

from app insights: 
<img width="963" alt="Screenshot 2024-06-13 at 4 58 42 PM" src="https://github.com/BiblioNexusStudio/well-web/assets/6589514/c41cce81-ee95-4771-811f-4f700bc933d0">